session_file.go

```diff
@@ -0,0 +1,133 @@
+package dtls
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"log"
+	"os"
+	"path"
+	"time"
+)
+
+// FileSessionStore is a simple file based SessionStore.
+// You need set a root path to store the session data.
+// And you can set an optional TTL to avoid long time session.
+//
+// FileSessionStore only clean session while fetching.  If you
+// want clean more aggressively, you could call the Clean() func.
+type FileSessionStore struct {
+	// Root store the session dir root path.
+	Root string
+	// TTL store the session store time duration.
+	TTL time.Duration
+}
```

Comment:
Storing plain master key to the disk basically breaks TLS's forward secrecy.
I think this implementation is nice for an example, but not suitable for product use.

Reply:
So, I would suggest to put `FileSessionStore` under `examples` directory.

Reply:
The `FileSessionStore` is just for demo or testing only. And should not be used in production code. Should we add this notice?

Reply:
Or should we offer another pure memory implementation?

Reply:
I think putting it under `./examples` directory with the notice is fine. Currently, it is under the root dir of the project, which usually means product ready code.